Added Configuration files for running in Maestro HPC at Pasteur Institute, Paris (#279)
nf-core/configs: docs/maestro.md

# nf-core/configs: Institut Pasteur Configuration on Maestro.

If you are using TARS cluster, please refer to pasteur profile.

This is an intent for configuration of nf-core in Maestro cluster on the normal or long qos depending on the ressources required

> I am not expert of nf-core so pelase refer to  docs/pasteur.md for installing and running nf-core 

Reviewer: ```suggestion
Please see the PASTEUR profile for installing and running nf-core instructions.
```

# nf-core/configs: Maestro (at Pateur Institute, Paris) Configuration.

To use, run the pipeline with -profile maestro,<qos> (with qos being long or normal). This will download and launch the maestro.config which has been pre-configured with a setup suitable for the Maestro cluster on either the long or normal qos.
Using one of these profiles, a docker image containing all of the required software will be downloaded, and converted to a Singularity image before execution of the pipeline

> 

Reviewer: ```suggestion
```
